demo/index.ts

```diff
@@ -569,7 +569,7 @@ dtmfForm?.addEventListener(
             return
         }
 
-        const callsInActiveRoom = Object.values(openSIPSJS.getActiveCalls).filter((call) => call.roomId === openSIPSJS.currentActiveRoomId)
+        const callsInActiveRoom = (Object.values(openSIPSJS.getActiveCalls) as Array<ICall>).filter((call) => call.roomId === openSIPSJS.currentActiveRoomId)
```

Comment:
Why use `as Array<ICall>`, Object.values is a generic function, it already returns you such type:

```typescript
interface ObjectConstructor {
    values<T>(o: { [s: string]: T } | ArrayLike<T>): T[];
}
```